File: LV0/연속된수의합/solution.js
function solution(num, total) {
    var answer = [];
    var first = (total/num)-((num-1)/2)
    
    for(var i =0; i<num; i++){
        answer[i]= first;
        first++;
    }
    return answer;
}

//https://school.programmers.co.kr/learn/courses/30/lessons/120923

File: LV0/연속된수의합/solution.py
def solution(num, total):
    answer = []
    first = (total/num)-((num-1)/2)
    i=0
    while i<num :
        answer.append(first)
        first+=1
        i+=1
    return answer

#https://school.programmers.co.kr/learn/courses/30/lessons/120923?language=python3